src/app/child-window-handler.ts

```diff
@@ -9,12 +9,7 @@ import { getGuid } from '../common/utils';
 import { config } from './config-handler';
 import { monitorWindowActions, removeWindowEventListener } from './window-actions';
 import { ICustomBrowserWindow, windowHandler } from './window-handler';
-import {
-    getBounds,
-    handleCertificateProxyVerification,
-    injectStyles,
-    preventWindowNavigation,
-} from './window-utils';
+import { getBounds, handleCertificateProxyVerification, injectStyles, preventWindowNavigation } from './window-utils';
```

Comment:
This change is not required

Reply:
I think it's because of the formatting. Do we have a formatting configuration for `TS`?